tox.ini

```
@@ -1,6 +1,7 @@
 [tox]
 minversion = 2.4
 envlist = lint,types,py{36,37,38,39},integration,docs
+isolated_build = True
```

This setting is the only one I don't recognize. This setting requires tox 3.3, so minversion above should be updated.

```suggestion
minversion = 3.3
envlist = lint,types,py{36,37,38,39},integration,docs
isolated_build = True
```

What is the value of this setting? Does pip not already do isolated builds by default for projects with pyproject.toml? If the default behavior is adequate, I'd be inclined to stick with the default behavior.

---

Oh, gawd. So I tried removing `isolated_build = True` and I got an error:

```
ERROR: pyproject.toml file found.
To use a PEP 517 build-backend you are required to configure tox to use an isolated_build:
https://tox.readthedocs.io/en/latest/example/package.html
```

Yuck. Tox has an optional setting that if you don't set will fail. What a terrible experience. I wonder why I haven't encountered this before. I have hundreds of projects with pyproject.toml running under tox without this setting.

---

Oh, probably because I use `use_develop=True` to avoid having to rebuild on each change.